src/Transport/Topology/NumberOfTopicsInBundleCheck.cs

```diff
@@ -9,6 +10,7 @@ static class NumberOfTopicsInBundleCheck
         public static async Task<NamespaceBundleConfigurations> Run(IManageNamespaceManagerLifeCycleInternal manageNamespaceManagerLifeCycle, NamespaceConfigurations namespaceConfigurations, string bundlePrefix)
         {
             var namespaceBundleConfigurations = new NamespaceBundleConfigurations();
+            var topicInBundleNameRegex = new Regex($@"^{bundlePrefix}\d+$", RegexOptions.CultureInvariant);
```

Comment:
Since we only ever execute this once it doesn't make sense to cache this regex, right?

Reply:
Correct @danielmarbach.
The time it will take to compile might exceed the execution time. I thought of caching it, but as you've outlined, we only do it once.